src/gateways/createBookConfigLoader.ts

```diff
@@ -0,0 +1,37 @@
+import { BOOKS } from '../config';
+
+interface BookVersion {
+  defaultVersion: string;
+}
+
+interface BookConfig {
+  [key: string]: BookVersion;
+}
+
+interface ReleaseJsonStructure {
+  books: BookConfig;
+  code: string;
+  id: string;
+}
+
+export default (baseUrl: string) => {
+  const loadBookRemoteBookConfig = (url: string): Promise<BookConfig | undefined> => {
+    return fetch(url)
+      .then((response) => {
+        if (response.status === 200) {
+          return response.json() as Promise<ReleaseJsonStructure>;
+        }
+      })
+      .then((response) => response && response.books)
+      .catch(() => {
+        return Promise.resolve(undefined);
```

Comment:
it would be nice to have this put a warning in sentry, but definitely not pop an error to the user